src/js/controller/controller.js

```diff
@@ -358,19 +358,30 @@ Object.assign(Controller.prototype, {
             }
         }
 
+        function _pauseWhenNotViewable(viewable) {
+            if (!viewable) {
+                _this.pause({ reason: 'viewable' });
+                _model.set('playOnViewable', !viewable);
```

Comment:
Are we just setting this to `true`?

Reply:
Essentially but for some reason, Rob wanted me to set it to `!viewable`. I don't remember why because it was a while ago.